adapters/yieldlab/yieldlab.go

```diff
@@ -344,6 +352,72 @@ func (a *YieldlabAdapter) makeCreativeID(req *openrtb_ext.ExtImpYieldlab, bid *b
 	return fmt.Sprintf(creativeID, req.AdslotID, bid.Pid, a.getWeek())
 }
 
+// unmarshalSupplyChain makes the value for the schain URL parameter from the openRTB schain object.
+func unmarshalSupplyChain(req *openrtb2.BidRequest) *openrtb2.SupplyChain {
+	var extSChain openrtb_ext.ExtRequestPrebidSChain
+	err := json.Unmarshal(req.Source.Ext, &extSChain)
+	if err != nil {
+		// req.Source.Ext could be anything so don't handle any errors
```

Comment:
I'm going to confirm with the team that this ignoring of error handling is okay. But in most circumstances, we usually try to avoid non-error handling.

Reply:
req.Source.Ext is a freeform field, so we can not be sure that customers will use it to pass the supply chain parameter. This will change with OpenRTB 2.6. There is a specified field for it.